Deep_Learning/HoneypotAI/honeypot/ssh_server.py

```
@@ -0,0 +1,202 @@
+"""
+SSH Honeypot Server
+Simulates SSH service to capture brute force and other SSH-based attacks
+"""
+
+import socket
+import threading
```

Ruff (0.12.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)